include/Gaffer/Private/IECorePreview/LRUCache.h

```diff
@@ -146,6 +146,8 @@ class LRUCache : private boost::noncopyable
 		/// Returns the current cost of all cached items.
 		Cost currentCost() const;
 
+		static void nullRemovalCallback( const Key &key, const Value &value );
```

Comment:
Addressed in ff178a4.  Note that this required adding an explicit check for empty callbacks, and that I've now removed nullRemovalCallback since it was private and unused.